templates/iam_policy/ingest_validate_generic_ingest_inputs_policy.json.tpl

```diff
@@ -0,0 +1,31 @@
+{
+  "Statement": [
+    {
+      "Action": [
+        "s3:GetObject",
+        "s3:HeadObject",
```

Comment:
There isn't an action for `s3:HeadObject`, it uses `s3:GetObject` - https://docs.aws.amazon.com/AmazonS3/latest/API/API_HeadObject.html.

AWS API endpoints != IAM actions 😵